setup.cfg

```diff
@@ -1,6 +1,5 @@
 [metadata]
 name = pyhf
-version = 0.6.1
```

Comment:
@kratsg @henryiii How does PyPI know the version of the release? Or is this just taken from the file name itself?

Reply:
It's added to the metadata by setuptools_scm.